crates/parcel_wasm/src/function_ref.rs

```diff
@@ -0,0 +1,31 @@
+use napi::Env;
+use napi::JsFunction;
+use napi::Ref;
+
+pub struct FunctionRef {
+  pub env: Env,
+  pub reference: Ref<()>,
+}
+
+// We don't currently call functions from multiple threads, but we'll need to change this when we do.
+unsafe impl Send for FunctionRef {}
```

Comment:
Why do you need this if you don't call it from multiple threads?

Reply:
Good point, I'm just moving existing code and not making changes to the sources